action.yml

```diff
@@ -18,6 +18,13 @@ inputs:
       Whitespace-separated fully-qualified image names.
       Each tag must be in the format: <registry>/[<namespace>/]<repository>:<tag>
     required: true
+  
+  smoke_test:
+    description: > 
+      Bash shell script to run as a smoke test against the built image.
+      Inside the script, you will have access to a variable called 'IMAGE_NAME' 
+      which you can use in your test to run the image.
+    required: true
```

Comment:
For backwards compatibility I don't think we should require this field. That means we can roll out the option to everyone by updating the `v1` tag, without breaking any existing usages of the action.

Backwards compatibility is valuable here because it means we'll still only really have one "release" version of the action to maintain. So all bugfixes etc will be enjoyed by all users without us having to backport them to older versions.

Eventually once we have enough reasons to, we may want to release a `v2` and at that point we could make this kind of breaking change along with any others we want.

Reply:
good point, I will change this to false.